test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js

```diff
@@ -1085,6 +1085,30 @@ describe('test/unit/lib/plugins/aws/deploy/lib/checkForChanges.test.js', () => {
     expect(serverless.service.provider.shouldNotDeploy).to.equal(true);
   });
 
+  it.only('should print a warning if missing lambda:GetFunction permission', async () => {
+    const { stdoutData } = await runServerless({
+      fixture: 'checkForChanges',
+      cliArgs: ['deploy'],
+      env: { AWS_CONTAINER_CREDENTIALS_FULL_URI: 'ignore' },
```

Comment:
Is that ENV variable needed in this case?

Reply:
No you are right I removed it thanks !